scripts/kernbench/cb_run_kernbench.sh

```diff
@@ -0,0 +1,60 @@
+#!/usr/bin/env bash
+
+cd ~
+
+source $(echo $0 | sed -e "s/\(.*\/\)*.*/\1.\//g")/cb_common.sh
+
+set_load_gen $@
+
+LOAD_PROFILE=$(echo ${LOAD_PROFILE} | tr '[:upper:]' '[:lower:]')
+
+KERNBENCH_NR_CPUS=$(get_my_ai_attribute_with_default kernbench_nr_cpus 0)
+
+# Override the default 4*cpu for make -j <nr>
+if test $KERNBENCH_NR_CPUS = 0; then
+	NRJOBS=""
+else
+	NRJOBS="-o $KERNBENCH_NR_CPUS"
+fi
+
+#If more than one command is needed (e.g., connected by "&&" or ";", please dump it on script, instead of just assigining to the variable CMDLINE"
+echo "#!/usr/bin/env bash" > kernbenchloadgen.sh
+echo "cd linux" >> kernbenchloadgen.sh
+echo "../ltp/utils/benchmark/kernbench-0.42/kernbench -n 1 -H -M -f $NRJOBS" >> kernbenchloadgen.sh
```

Comment:
Here, we want to change this to use ${dir} so that is relative to the cbtool installation directory.